api/__init__.py

```diff
@@ -27,6 +30,7 @@ def create_app(config_name=None):
     api.init_app(app)
     ext_celery.init_app(app)
     db.init_app(app)
+    thc.init_app(app)
```

Comment:
This way of instantiating `ToolhubClient` couples it to the specific implementation –  what if you tried to use this Class outside the app context? This mimics how Flask extensions work, but do we want that? Flask extensions need the app context, and they also need to keep track of state. `ToolhubClient` only needs access to the Toolhub URL, and the request headers. It would be okay to have several instances of this class, each created in the context where it's used.

Reply:
That's true, and it might solve a problem that I'm having with the Celery tasks.  It probably won't, but it's worth a shot.